CHANGELOG.md

```diff
@@ -18,6 +22,7 @@
 * [#2255](https://github.com/bbatsov/rubocop/issues/2255): Refine the offense highlighting for `Style/SymbolProc`. ([@bbatsov][])
 * [#2260](https://github.com/bbatsov/rubocop/pull/2260): Make `Exclude` in `.rubocop_todo.yml` work when running from a subdirectory. ([@jonas054][])
 
+
```

Comment:
You should probably remove the extra blank line. Not sure how it got here, but I guess it was by accident.